os/tools/mkbinheader.py

```diff
@@ -102,10 +108,65 @@ def get_static_ram_size(bin_type):
     os.remove(STATIC_RAM_ESTIMATION)
     return static_ram_size
 
+############################################################################
+#
+# Kernel binary header information :
+#
+# total header size is (16 + #)bytes.
+# +-----------------------------------------------------------------------------------
+# | Header size | Binary Version |  Binary Size | Secure Header Size | Secure Header |
+# |   (4bytes)  |    (4byte)     |    (4byte)   |       (4byte)      |    (#bytes)   |
```

Comment:
In user binary header, header size is 2bytes. But for kernel, it is 4bytes.
I think 2bytes is also enough to kernel. What do you think about?